lighthouse-core/index.js

```diff
@@ -48,6 +48,11 @@ async function lighthouse(url, flags = {}, configJSON, connection) {
   flags.logLevel = flags.logLevel || 'error';
   log.setLevel(flags.logLevel);
 
+  const channelOverride = /** @type {any} */(process.env.LH_CHANNEL) || flags.channel;
```

Comment:
do we have an immediate need to have a `process.env` override? If not, let's wait to add it

Reply:
I'm guessing Matt does.

Reply:
I believe I originally made this request in the old PR for WebPageTest

Reply:
Yeah just added it because the old PR mentioned wpt.

Reply:
WPT makes sense. If we want to let folks override it, should it just be a regular cli/node flag (overriding only if not set)?

The type will also need to widen to just `string`, since we don't want anyone relying on it being a small set of strings when it could be anything.

Reply:
Have removed the `LH_CHANNEL` env logic, and added a channel CLI flag with a default value of `cli`.